<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2023</width>
    <height>1108</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1" rowspan="4">
     <widget class="QGroupBox" name="ResultView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="ClearpushButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SavepushButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PrintpushButton">
           <property name="text">
            <string>Print</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NamelineEdit"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Reg Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Previous CGPA?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CGPAcomboBox"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Number of Courses</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CoursescomboBox"/>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="CoursespushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Go to Courses</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Courses</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_0">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_1">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_2"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_3"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_6"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_4"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_7"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_5"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>560</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_13"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_10"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>520</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_12"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_8"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>480</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_11"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>600</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_14"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="inputLayout_9"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_26">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>790</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_39"/>
      </widget>
      <widget class="QPushButton" name="CalculatepushButton">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>530</y>
         <width>106</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_9">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>40</y>
         <width>391</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="cuandsplayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2023</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
